### PR TITLE
Improve getting IP address of VM

### PR DESCRIPTION
**Pull request by thomas-crowe**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New config Pull Request
- New role Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by thomas-crowe** [comment]
Adding 'ignore_errors' due to task failing when no IPv4 address was found because it had not been assigned yet.

**Review by jkupferer — CHANGES_REQUESTED** [review]
`ignore_errors` does not seem appropriate here.

**Review by bbethell-1 — APPROVED** [review]
LGTM

File: ansible/roles_ocp_workloads/ocp4_workload_mad_roadshow/tasks/rhv-setup-oracle-vm.yml [hunk 1]
```diff
@@ -20,17 +20,15 @@
   ovirt.ovirt.ovirt_nic_info:
     auth:
       insecure: true
-    vm: "{{ ocp4_workload_mad_roadshow_oracle_vm_name }}"
+    vm: "{{ ocp4_workload_ama_demo_oracle_vm_name }}"
```
Comment (jkupferer) [inline ansible/roles_ocp_workloads/ocp4_workload_mad_roadshow/tasks/rhv-setup-oracle-vm.yml:23]: Is this supposed to change here?

**Review by jkupferer — APPROVED** [review]

**Review by bbethell-1 — APPROVED** [review]

**Pull request merged**